### PR TITLE
Add additional example .mol files to inst/extdata

### PR DESCRIPTION
**Pull request by Aariq**
- Replaces and adds .mol files to inst/extdata as discussed in #79 
- Adds the names of the example compounds to the documentation for `mol_example()`

### CODE REVIEW

**Comment by Aariq** [comment]
Before I merge this, would it be better if `mol_example()` just returned a (named) vector of file paths?  E.g.

``` r
mol_example()
#> [1] "/Users/ericscott/Documents/GitHub/volcalc/inst/extdata/C00031.mol"
#> [2] "/Users/ericscott/Documents/GitHub/volcalc/inst/extdata/C08491.mol"
#> [3] "/Users/ericscott/Documents/GitHub/volcalc/inst/extdata/C16181.mol"
#> [4] "/Users/ericscott/Documents/GitHub/volcalc/inst/extdata/C16286.mol"
#> [5] "/Users/ericscott/Documents/GitHub/volcalc/inst/extdata/C16521.mol"
```

Then you could do things like `calc_vol(mol_example())`
@KristinaRiemer

**Comment by Aariq** [comment]
possibly add readLines() to example for mol_example()

**Review by KristinaRiemer — APPROVED** [review]

**Pull request merged**
